### ISSUE TITLE
Visual way to group components

### ISSUE THREAD

**Issue by AdRiley** [enso-org/enso#9519]
Add a new menu for a group of components equivalent to the wheel menu that we have for a single component.

![Image](https://github.com/enso-org/enso/assets/1720119/eb67a6b1-1b4e-4c31-9455-f881c2a9d954)

- The menu only appears when a group of components is selected.
- It will tell the user how many components have been selected to give them visual clue what the buttons will do
- Initially it will have 2 icons
- The group icon (to be supplied) is equivalent to pressing CTRL-G
- The palette icon opens the color picker and colors all selected component with the chosen color.

**Comment by farmaazon** [comment]
Refinement notes:
* The component visibility should be updated only after drag release (to avoid blinking).
* We should have `commited selection` set published in selection composable (and use it also for extending nodes).
* We should not consider case where there's no place for this new panel, because the small window does not work well anyway and need a separate issue.

**Comment by AdRiley** [comment]
Added new group icon to repo
